README.md

```diff
@@ -296,7 +296,7 @@ helm delete --namespace test my-application
 | certificate.duration | string | `"8760h0m0s"` | The requested "duration" (i.e. lifetime) of the Certificate. |
 | certificate.renewBefore | string | `"720h0m0s"` | The amount of time before the currently issued certificate's notAfter time that cert-manager will begin to attempt to renew the certificate. |
 | certificate.subject | tpl/object | `nil` | Full X509 name specification for certificate. |
-| certificate.commonName | string | `"admin-app"` | Common name as specified on the DER encoded CSR. |
+| certificate.commonName | string | `nil` | Common name as specified on the DER encoded CSR. |
```

Comment:
@owais-rehman maybe you can update the description and mention that it is not required/recommended for leaf certificates which this application CR presents